apps/vaporgui/MainForm.h

```diff
@@ -35,6 +35,7 @@
 #include <QIcon>
 #include <QLineEdit>
 #include <QWidgetAction>
+#include <chrono>
```

Comment:
Maybe Git is being funky here.  Could this be something from the progress bar branch?

Reply:
I had to add that to get it to compile on garibaldi